src/containers/Login/index.js

```diff
@@ -1,8 +1,10 @@
 import React, { Component } from 'react';
-import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
-import { LogInRequest } from '../../redux/actions/login';
 import LoginForm from '../../components/Login';
+import { setToken, removeToken } from '../../utils';
+import {
+  loginError, loginSuccess, removeMessage, LogInRequest,
```

Comment:
you can add these to `mapStateToProps` so you don't have to call `dispatch` explicitly

Reply:
yeah sure i meant  mapDispatchtoProps